plugins/notifiers/slack/client.go

```diff
@@ -67,11 +68,16 @@ func (n *notifier) Notify(items []domain.Notification) []error {
 	return errs
 }
 
-func (n *notifier) sendMessage(channel, text string) error {
+func (n *notifier) sendMessage(channel, messageBlock string) error {
 	url := slackHost + "/api/chat.postMessage"
-	data, err := json.Marshal(map[string]string{
+	var messageblockList []interface{}
+
+	if err := json.Unmarshal([]byte(messageBlock), &messageblockList); err != nil {
+		return err
```

Comment:
Would suggest to add some context to the errors returned here, so that its easier to debug which place the code gave out

Reply:
Done